snippets/buy-buttons.liquid

```diff
@@ -42,28 +42,30 @@
           class="product-variant-id"
         >
         <div class="product-form__buttons">
+          {%- liquid
+            assign check_against_inventory = true
+            if product.selected_or_first_available_variant.inventory_management != 'shopify'
+              assign check_against_inventory = false
+            elsif product.selected_or_first_available_variant.inventory_policy == 'continue'
+              assign check_against_inventory = false
+            endif
```

Comment:
No they shouldn't be able to as when they're not tracking the inventory through Shopify that `Continue selling when out of stock` checkbox is hidden like shown in the screenshot of my above comment 👍

Reply:
i think in Bulk Editor you still can.

Reply:
You're right 👍 Though I don't think it should be a problem with the current logic we have 🙂 
Cause if it's not tracked in Shopify, we won't disable the button. And if it's not tracked and also set to continue selling when out stock, it will still not disable the buy buttons 👌